docker-compose.yml

```diff
@@ -43,4 +66,8 @@ services:
       - "8888:8888"
     volumes:
       - nginx-shared:/nginx
-    command: docker/web/run_web.sh
\ No newline at end of file
+    command:
+      docker/web/wait_for_it.sh rabbitmq:5672 --timeout=30 --strict --
```

Comment:
You can use healthCheck in docker compose: https://medium.com/@saklani1408/configuring-healthcheck-in-docker-compose-3fa6439ee280

Reply:
Yes, bad decision. Updated [here](https://github.com/safe-global/safe-decoder-service/pull/10/commits/3612bf1cb83c7942bc9a114c0c40e18419168350)

Reply:
Not at all. I've just discovered you can do that a couple of months ago